index.js

```diff
@@ -183,12 +187,12 @@ module.exports = {
 			files: ['**/*.ts?(x)'],
 			parser: '@typescript-eslint/parser',
 			parserOptions: {
-				ecmaVersion: 2018,
+				ecmaVersion: 'latest',
```

Comment:
can you share the background information about this change?

Reply:
I forgot to undo this change. Thank you for noticing!